pacttesting/mock_service.go

```diff
@@ -162,6 +163,8 @@ func loadRunningServer(provider, consumer string) *MockServer {
 	server.Running = true
 	pactServers[provider+consumer] = &server
 	viper.Set(provider, server.BaseURL)
+	//Also set the base url as an environment variable to remove dependency on viper
+	os.Setenv("PACTTESTING_" + strings.ToUpper(provider), strings.Replace(server.BaseURL, "-", "_", -1))
```

Comment:
Please, handle errors as [`os.Setenv()` can fail](https://go.dev/play/p/jPbbkDpi-sy) and we should not just continue.